### ISSUE TITLE
🚨 BUILD BROKEN 🚨

### ISSUE THREAD

**Issue by mathiasmagnusson** [datasektionen/skywhale#68]
It tries to run apt update with some postgres ppa thing that doesn't have bionic anymore

**Comment by Herkarl** [comment]
This could be related? 
https://github.com/dokku/dokku/discussions/5936

**Comment by mathiasmagnusson** [comment]
Maybe, but I tried running `dokku plugin:update apt` and it just said we didn't have the apt plugin installed.
Perhaps updating the php buildpack or postgres plugin could work?

**Comment by mathiasmagnusson** [comment]
I also noticed that since we've tried to push new builds that don't work it tries use another image when restarting the app, so now we can't change environment variables. I also think that it would not start properly if the server has to restart.

I have however saved the currently running (and working) image in my home directory on the server and while I don't seem to be able to change environment variables in it (maybe I'm doing it at the wrong place), I have gotten it to run locally. In case we have to do this I'm writing some instructions on how below. Note that they're podman specific but easily translated to docker.

1. Get the image: `scp sips:/home/mathm/skywhale.tar.gz .`
2. Load the image: `podman load < skywhale.tar.gz` and rename `podman image tag sha256:whatever_hash_last_command_output_at_bottom skywhale:latest`
3. Create a pod: `podman pod create -p5000:5000 skywhale`
4. Create a database: `podman run --pod skywhale --name dokku-postgres-skywhale -d -e POSTGRES_USER=postgres -e POSTGRES_PASSWORD=the_password_found_in_the_image -e POSTGRES_DB=skywhale postgres`
5. Get data into the database from a dump from prod, or run migrations: `podman run --pod skywhale --rm --workdir /app skywhale:latest ./.heroku/php/bin/php artisan migrate`
6. Start the app with: `podman run --pod skywhale --init --name skywhale_web -d -e PORT=5000 --workdir /app skywhale:latest /start web`
